Code/Framework/AzToolsFramework/AzToolsFramework/Editor/ActionManagerUtils.h

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+namespace AzToolsFramework
+{
+    static constexpr const char s_actionManagerToggleKey[] = "/O3DE/ActionManager/EnableNewActionManager";
```

Comment:
nit: My only suggestion would be moving this to the .cpp to encourage people to just use the helper and not use this directly (if it's going to stay in the .h file we should mark it `inline` and not `static` too to avoid additional copies being made)